### PR TITLE
Migrate basic build to drone

### PR DESCRIPTION
**Pull request by Arrow768**
Migrate a very basic build to drone, so we have at least some information if a PR compiles and passes the UTs or not.

### CODE REVIEW

File: code/world.dm [hunk 1]
```diff
@@ -349,13 +349,16 @@ var/list/world_api_rate_limit = list()
...
+	if(!config.sql_enabled)
+		world.log << "Database Connection disabled. - Skipping Connection Establishment"
+		return 1
 	//Construct the database object from an init file.
 	dbcon = initialize_database_object("config/dbconfig.txt")
 
-	if (!setup_database_connection(dbcon))
-		world.log <<  "Your server failed to establish a connection with the feedback database."
+	if(!setup_database_connection(dbcon))
+		world.log <<  "Your server failed to establish a connection with database."
```
Comment (Arrow768) [inline code/world.dm:359]:
```suggestion
		world.log <<  "Your server failed to establish a connection with the configured database."
```

File: scripts/code_check.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    ERROR_COUNT=$(($ERROR_COUNT+1))
+    echo "FAIL: to_world count $TO_WORLD_COUNT_FOUND does not match expected $TO_WORLD_COUNT" >> code_error.log
+else
+    echo "PASS: to_world count matches" >> code_error.log
+fi
+
+echo "Found $ERROR_COUNT Errors while performing code check"
+
+if [ $ERROR_COUNT -ne 0 ]; then
+    echo "Printing code_error log"
```
Comment (Arrow768) [inline scripts/code_check.sh:80]:
```suggestion
```

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    image: mariadb:10.3
+    commands:
+      - sleep 20
+      - for f in SQL/migrate/*.sql; do mysql -h mariadb -uroot -proot game < $f && echo "Migrated $f"; done
+      - mysql -h mariadb -uroot -proot game -e "SHOW TABLES;"
+
+  - name: "Runtime - Compile and Run Unit Tests" 
+    image: aurorastation/dm-buildimage:latest
+    commands:
+      - export LD_LIBRARY_PATH=./:$PWD:$HOME/.byond/bin:/usr/local/lib:$LD_LIBRARY_PATH
```
Comment (skull132) [inline .drone.yml:67]: Shouldn't this technically be done in the image? Or will the session not persist?

**Review by skull132 — APPROVED** [review]

**Comment by Arrow768** [comment]
In it goes then. The requirements installation will be tweaked in a later update anyway.

**Pull request merged**